alphabase/constants/modification.py

```diff
@@ -376,11 +406,12 @@ def _add_a_new_modification(
             "modloss_composition",
             "classification",
             "unimod_id",
+            "smiles",
         ],
-    ] = [mod_name, composition, modloss_composition, "User-added", 0]
+    ] = [mod_name, composition, modloss_composition, "User-added", 0, smiles]
     MOD_DF.loc[mod_name, ["mass", "modloss"]] = (
-        calc_mass_from_formula(composition),
-        calc_mass_from_formula(modloss_composition),
+        composition_mass,
+        modloss_mass,
```

Comment:
(nit) why introduce variables here? at least, I would move them right above this line ;-)